test/ui-test/testSuites/suite_status/tst_ChatFlow/test.feature

```diff
@@ -50,6 +50,13 @@ Feature: Status Desktop Chat
          Then the user can delete the message at index 0
          Then the last message is not the random message
 
+    Scenario: The user is able to use emoji suggestions
+         When user joins chat room automation-test
+		 Then user is able to send chat message
+		 | message  			      |
+		 | Hello :thumbs   			 |
+		 Then the user can select emoji
```

Comment:
I'd suggest to split into action and validation so:
```suggestion
		 When the user selects emoji in the suggestion list
		 Then the last message is XX 
		 OR
		 Then the last message contains the emoji XX  
```
So the validation you are now doing `verify_text_contains(str(textInput.text), str(":thumbsup:"))` I'd keep it in the first `When` statement, but then extend the validation to see if it can be really sent.